src/mixins/form-radio-check-group.js

```diff
@@ -85,12 +85,12 @@ export default {
           key: uid,
           props: {
             id: this.safeId(uid),
-            value: option.value,
-            disabled: option.disabled || null
-            // Do we need to do these, since radio's will know they are inside here?
+            value: option.value
+            // We don't need to include these, since the input's will know they are inside here
+            // disabled: option.disabled || null,
```

Comment:
A single checkbox/radio can be disabled.... so this one shouldn't be commented out.

Reply:
When providing an array of options:

```js
options = [
 { text: 'option 1', value: 1},
 { text: 'option 2', value: 2},
 { text: 'option 3', value: 3, disabled: true },
 { text: 'option 4', value: 4}
]
```

Reply:
It is now 👍

Reply:
It shouldn't be commented out.

As a user could have a list of 5 checkboxes (provided by the options array), and one of those individual checkboxes can be disabled (even if the group of checkboxes is not disabled)

Reply:
The checkbox will look to see if it is disabled or if the group is disabled and set it's disabled attribute if either one of those evaluate to true.

Reply:
`b-form-select` also uses the options mixin, and supports disabling an individual option value

Reply:
See the options docs at: https://deploy-preview-2721--bootstrap-vue.netlify.com/docs/components/form-select#options-property

Reply:
With this options array:
```js
        options: [
          { text: 'Orange', value: 'orange' },
          { text: 'Apple', value: 'apple', disabled: true },
          { text: 'Pineapple', value: 'pineapple' },
          { text: 'Grape', value: 'grape' }
        ]
```

In current release:
![image](https://user-images.githubusercontent.com/2781561/53456021-14376b80-3a03-11e9-9457-bce8e467a003.png)

Which shows the option `Apple` disabled

In this PR:
![image](https://user-images.githubusercontent.com/2781561/53456030-1f8a9700-3a03-11e9-91c8-35559cfdf6c8.png)

Which shows the option `Apple` not respecting it's disabled state.

Reply:
So this PR is introducing an issue (breaking change) with not respecting the options array `disabled` properties.

Reply:
I've updated the test suites to test when an option in the options array is set to `disabled: true`.

Reply:
I've uncommented the `disabled` prop passing from the options array.